### PR TITLE
LPS-161440 Stabilize frontend test suite

### PR DESCRIPTION
**Pull request by john-co**
**Description**
Only run stable FI tests in frontend test suite since we've been seeing too many false positives.

**Test Plan**
- [x] non-quarantined tests owned only by FI team are run in `ci:test:frontend`

**JIRA TICKET**
https://issues.liferay.com/browse/LPS-161440

### CODE REVIEW

**Comment by john-co** [comment]
ci:stop

**Comment by john-co** [comment]
ci:test:sf

**Comment by john-co** [comment]
ci:test:relevant

**Comment by john-co** [comment]
ci:test:frontend

**Comment by john-co** [comment]
ci:stop

**Comment by john-co** [comment]
ci:test:frontend

**Comment by john-co** [comment]
ci:test:relevant

**Review by susanchen3 — APPROVED** [review]
makes sense to me :)

File: test.properties [hunk 1]
```diff
@@ -4134,24 +4134,23 @@
...
     test.batch.run.property.query[functional-tomcat90-mysql57-jdk8][frontend]=\
-        (portal.upstream == true) AND \
         (\
-            (testray.component.names ~ "Cadmin") OR \
-            (testray.component.names ~ "Frontend Dataset") OR \
-            (testray.component.names ~ "Remote Apps") OR \
-            (testray.component.names ~ "Theme") OR \
-            (testray.component.names ~ "Translation Manager") OR \
-            (testray.component.names ~ "WYSIWYG") OR \
-            (testray.component.names ~ "Walkthrough") OR \
```
Comment (yanan) [inline test.properties:4145]: Just wondering these removed components no longer belong to frontend anymore?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteAppsUpgrade.testcase [hunk 2]
```diff
@@ -59,6 +56,9 @@ definition {
 	@priority = "5"
 	test CanViewRemoteAppFieldsAfterUpgrade73101 {
 		property data.archive.type = "data-archive-remote-app";
+		property portal.acceptance = "false";
+		property portal.release = "false";
+		property portal.upstream = "quarantine";
```
Comment (yanan) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteAppsUpgrade.testcase:61]: Thanks for the changes!

File: test.properties (hunk 1 above)
Comment (john-co) [reply test.properties:4145]: The component.names can be applied on tests that are not owned by FI team, they don't necessarily belong to FI. ie: On BlogsAdmin#CanAddEntryInAdmin it has `testray.component.names ~ "WYSIWYG"`. Test still owned by Lima team, but I placed WYSIWYG component for `ci:test:relevant` purposes to target usecase tests for ckeditor module.

On Testray, I believe it uses  `testray.main.component.name` to filter by `Team` and would better define which tests belong to frontend infra.

**Comment by john-co** [comment]
Thanks @susanchen3 and @yanan !

Looks like there's a regression on Remote Apps... looks like caused by https://issues.liferay.com/browse/LPS-159174

File: test.properties (hunk 1 above)
Comment (yanan) [reply test.properties:4145]: Good to know, thanks so much for the explanation!

**Review by yanan — APPROVED** [review]

**Comment by john-co** [comment]
ci:forward

**Pull request closed without merging**